### PR TITLE
Improve docs for sql env vars

### PR DESCRIPTION
**Pull request by johnsimons**

### CODE REVIEW

File: src/ServiceControl.Transports.SqlServer/SqlServerQuery.cs [hunk 1]
```diff
@@ -147,8 +148,12 @@ protected override void InitialiseCore(FrozenDictionary<string, string> settings
     public static class SqlServerSettings
     {
         public static readonly string ConnectionString = "SqlServer/ConnectionString";
-        public static readonly string ConnectionStringDescription = "A single database connection string that will provide at least read access to all queue tables.";
+
+        public static readonly string ConnectionStringDescription =
+            "Database connection string that will provide at least read access to all queue tables.";
```
Comment (jpalac) [inline src/ServiceControl.Transports.SqlServer/SqlServerQuery.cs:153]: If there's no database specified (ie it's not a "single" db connection) then how would SC know which database/catalog to use?

Comment (johnsimons) [reply src/ServiceControl.Transports.SqlServer/SqlServerQuery.cs:153]: it would error, and not start

Comment (johnsimons) [reply src/ServiceControl.Transports.SqlServer/SqlServerQuery.cs:153]: The connection string requires the initial catalog to be set

Comment (jpalac) [reply src/ServiceControl.Transports.SqlServer/SqlServerQuery.cs:153]: So why remove the word "single" from the description then?

Comment (johnsimons) [reply src/ServiceControl.Transports.SqlServer/SqlServerQuery.cs:153]: because there is no "multiple"

**Review by jpalac — APPROVED** [review]
Can you please make the same updates on the params in docs PR?

**Comment by johnsimons** [comment]
> Can you please make the same updates on the params in docs PR?

already done that

**Pull request merged**